<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>最短路径课程设计</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算最短路径</string>
   </property>
  </widget>
  <widget class="PaintWidget" name="paintWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>711</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起    点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>终    点</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>通达</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>扬大</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>汇金谷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>顺达</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三湾湿地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>万达</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宝龙</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>京华城</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三盛</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>荷花池</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>火车站</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>文昌阁</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>凯德</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双博馆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>东关街</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>茱萸湾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>瘦西湖</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>通达</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>扬大</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>汇金谷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>顺达</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三湾湿地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>万达</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宝龙</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>京华城</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三盛</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>荷花池</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>火车站</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>文昌阁</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>凯德</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双博馆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>东关街</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>茱萸湾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>瘦西湖</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最短路径课程设计</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header location="global">paintwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
